src/storage/admin/RebuildIndexTask.cpp

```diff
@@ -17,6 +17,17 @@ const int32_t kReserveNum = 1024 * 4;
 bool RebuildIndexTask::check() {
   return env_->kvstore_ != nullptr;
 }
+void RebuildIndexTask::finish(nebula::cpp2::ErrorCode rc) {
```

Comment:
why just add check if it is IndexState::FAILED in line 76? If it is, then could skip it.